base/multidimensional.jl

```diff
@@ -146,13 +146,55 @@ module IteratorsMD
         return h
     end
 
-    # nextind and prevind with CartesianIndex
+    """
+        nextind(a::AbstractArray{<:Any,N}, i::CartesianIndex{N}) where {N}
```

Comment:
Can this be combined with the `i::Integer` docs?   i.e. have a docstring for `nextind(a::AbstractArray, i)`?

(When I do `?nextind`, having docstrings for a number of nearly identical methods scroll by makes the documentation harder to read.)

Reply:
Sure, fixed in 233dc9ba21.